<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DS-C101AA553646" name="com.cil.ui.TestCILPortal" tests="10" failures="1" timestamp="29 Dec 2017 11:27:32 GMT" time="103.780" errors="0">
  <testcase name="VerifyToolsAndTechnologiesOnDashboard" time="10.628" classname="com.cil.ui.TestCILPortal"/>
  <testcase name="VerifyOpinionPollSubmit" time="5.515" classname="com.cil.ui.TestCILPortal"/>
  <testcase name="VerifyTrendingIdeasOnDashboard" time="5.552" classname="com.cil.ui.TestCILPortal"/>
  <testcase name="VerifyCILDashboardComponents" time="10.141" classname="com.cil.ui.TestCILPortal"/>
  <testcase name="test7" time="5.030" classname="com.cil.ui.TestCILPortal">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]at org.testng.Assert.fail(Assert.java:96)at org.testng.Assert.failNotEquals(Assert.java:776)at org.testng.Assert.assertTrue(Assert.java:44)at org.testng.Assert.assertTrue(Assert.java:54)at com.cil.ui.CILPageMethods.VerifyTest7(CILPageMethods.java:105)at com.cil.ui.TestCILPortal.test7(TestCILPortal.java:44)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)at java.lang.reflect.Method.invoke(Unknown Source)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </failure>
  </testcase> <!-- test7 -->
  <testcase name="VerifyLabVisitsOnDashboard" time="13.678" classname="com.cil.ui.TestCILPortal"/>
  <testcase name="VerifyUpcomingEventsOnDashboard" time="11.338" classname="com.cil.ui.TestCILPortal"/>
  <testcase name="VerifyOnGoingProjects" time="26.650" classname="com.cil.ui.TestCILPortal"/>
  <testcase name="VerifyLinksOnCILDashboard" time="5.235" classname="com.cil.ui.TestCILPortal"/>
  <testcase name="VerifyLinksOnCILHomePage" time="10.013" classname="com.cil.ui.TestCILPortal"/>
</testsuite> <!-- com.cil.ui.TestCILPortal -->
